styleguide/progress.scss

```diff
@@ -0,0 +1,46 @@
+@import 'colors';
+@import 'typography';
+
+// progress bars
+.kiln-progress-wrapper {
+  bottom: 100%;
```

Comment:
And therefore not defending your space.  Using negative space like this means adding anything around your DOM element is going to be very difficult in the future.